mbed_app.json

```diff
@@ -161,6 +184,81 @@
             "target.features_remove"                   : ["CRYPTOCELL310"],
             "target.macros_remove"                     : ["MBEDTLS_CONFIG_HW_SUPPORT"],
             "target.macros_add"                        : ["NRFX_RNG_ENABLED=1", "RNG_ENABLED=1", "NRF_QUEUE_ENABLED=1"]
+        },
+        "NUMAKER_PFM_NUC472": {
```

Comment:
I would prefer that you create a new config file for boards which use an out-of-tree SD card driver. @SeppoTakalo What is your opinion in this matter?

Reply:
> Other option, assumably, would be to use ""target.components_remove"" for these boards.

Build tool doesn't allow `"target.components_add": ["SD"],` in `*` and then `"target.components_remove": ["SD"],` in e.g. NUMAKER_PFM_NUC472. So move the line `"target.components_add": ["SD"],` from `*` to every target needing it.

```
[ERROR] Configuration conflict. The component SD both added and removed.
```